calculator.py

```diff
@@ -16,10 +18,10 @@ def total_force(velocity, throttle_force):
     return throttle_force - air_friction - SLIDING_FRICTION
 
 
-def velocity(old_velocity, throttle_force):
+def velocity(old_velocity, previous_timestamp, throttle_force=THROTTLE_FORCE):
     acceleration = total_force(old_velocity, throttle_force) / CAR_MASS
-    velocity = acceleration + old_velocity
-    return velocity if velocity >= 2 else 0
+    velocity = old_velocity + acceleration * (time.time() - previous_timestamp)
+    return velocity if velocity > 0 else 0
```

Comment:
Add maybe a comment behind it to indicate that the if-statements prevents a negative velocity.